pyright_reportGeneralTypeIssues.json

```diff
@@ -0,0 +1,127 @@
+# this becomes obsolete when reportGeneralTypeIssues can be enabled in pyproject.toml
```

Comment:
Can be a followup if it makes sense, but would it appropriate to move this in the `typing` directory?

Reply:
Could move it there or to `ci/`. I think `typings` is meant for type-stubs for external libraries that have no/missing type annotations (numba).

Reply:
> I think `typings` is meant for type-stubs for external libraries that have no/missing type annotations (numba).

that's correct. it's in the default search path for type stubs and putting stubs there override stubs from imports.

In time, we shouldn't need the directory when our dependencies have complete type stubs.

so I would advise against putting anything other than stubs there.

Reply:
Got it thanks. Fine where it is then.